### DISCUSSION TITLE
Image to PDF resulting in huge file sizes?

### DISCUSSION THREAD

**Discussion by welles** [Stirling-Tools/Stirling-PDF#213] [Q&A] [answered]
Hi, for some reason when I use Image to PDF it results in a PDF that is around six times the file size of all the input images combined and I don't understand why.
Like I was trying to combine two images with around 3.5MB each to a PDF and it resulted in a file that was 44MB big.
Is there a bug here? I can't imagine that this could have a legitimate reason.

**Comment by Frooodle** [comment]
Hmm are your images jpg or png

I need to check what formats I embedded them into pdf with but could be I am converting by mistake to some lossless version  will check code later

**Reply by welles** [reply]
JPG. I'll try to get you some example files. Thanks for looking into this!

**Reply by welles** [reply]
Here are two example pages I just scanned. Combining them in Image to PDF results in a 55MB file, even though the images are around 8MB combined.

![IMG_0001](https://github.com/Frooodle/Stirling-PDF/assets/10435927/00bce7e0-aa60-4fb5-bf19-f91a484e6b6d)
![IMG_0002](https://github.com/Frooodle/Stirling-PDF/assets/10435927/4b66ad18-1a81-4cff-83f6-5d7228a6d4c2)

**Reply by Frooodle** [reply, marked as answer]
Just checked code
I see I convert any image passed in into a lossless format
This is causing jpegs to become huge in filesize
I'll add some automated flag to keep it lossy (jpeg) if it detects that type of image format
Should help a lot with this

**Reply by Frooodle** [reply]
This was fixed in latest release 
Let me know if it helped at all

**Reply by welles** [reply]
Thanks! Just tested it, the file is now the correct size!
